packages/barracuda/0.1.0/dataset/waf/agent/stream/stream.yml.hbs

```diff
@@ -0,0 +1,3721 @@
+paths:
+{{#each paths as |path i|}}
+  - {{path}}
+{{/each}}
+exclude_files: [".gz$"]
+tags: {{tags}}
+fields_under_root: true
+fields:
+    observer:
+        vendor: "Barracuda"
+        product: "Web"
+        type: "WAF"
+
+publisher_pipeline.disable_host: true
+
+processors:
+- script:
+    lang: javascript
+    params:
+      ecs: true
+      rsa: {{rsa_fields}}
+      tz_offset: {{tz_offset}}
+      keep_raw: {{keep_raw_fields}}
+      debug: {{debug}}
+    source: |
+      // Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
@nchaulet @michalpristas Could you please confirm that such huge blobs should be supported by all parties?